traffic_cam/paths.py

```diff
@@ -8,3 +8,9 @@
 CLASSIFIER_DIR = Path("data/classifier/")
 CLASSIFIER_HDF5 = CLASSIFIER_DIR / "model.hdf5"
 CLASSES_JSON = CLASSIFIER_DIR / "class_indices.json"
+
+# Ensure paths
+DATA_DIR.mkdir(parents=True, exist_ok=True)
+TRAIN_DIR.mkdir(parents=True, exist_ok=True)
+
+CLASSIFIER_DIR.mkdir(parents=True, exist_ok=True)
```

Comment:
could we remove these commands from paths.py?
I would be uncomfortable to import from this file, if that also executes code.

Reply:
Changed to a function which needs to be called on-demand